node_modules/oae-content/tests/test-collabdoc.js

```diff
@@ -489,6 +489,48 @@ describe('Collaborative documents', function() {
     });
 
     /**
+     * Test that verifies that no new revisions are created when an etherpad document is published
```

Comment:
@mrvisser : You mentioned that your test error was caused by the default Etherpad text not being set to empty text. Should we have a test to validate that directly as well?